### PR TITLE
updating forum

### LINKED ISSUE #16: forum-middleware-update
file structure has been fixed, likely creating middleware for forum

### PR DESCRIPTION
**Pull request by ZhiLinR**
creating middleware for forum

### CODE REVIEW

**Comment by ZhiLinR** [comment]
it would appear that comment does not have a get method 🗿

**Comment by ZhiLinR** [comment]
fixes
- added a small middleware and util which formats and handles response
- GET post by id now includes comments (alternative is creating new route to fetch specifically from comments?

File: microsvcs/forum/app/controller/comments_con.js [hunk 1]
```diff
@@ -1,55 +1,52 @@
...
-        if (error.message == "Server Error Occurred") {
-            res.status(500).send(HANDLER.createErrorResponse(error.message));
+        if (result.deletedCount === 1) {
+            next(UTIL.formatRes(true, 200, "Comment Deleted"));
         } else {
-            res.status(404).send(HANDLER.createErrorResponse(error.message));
+            next(UTIL.formatRes(false, 404, "No Reference Found"))
         }
+    } catch (error) {
+        console.log(error)
```
Comment (brandontjq) [inline microsvcs/forum/app/controller/comments_con.js:49]: you can consider exploring better logging options for e.g. `express-winston`, which is a popular logging library for express

**Comment by brandontjq** [comment]
Only 1 comment that you can pick up in your next PR.  Rest of it LGTM :) Feel free to merge anytime

**Comment by ZhiLinR** [comment]
research point: logging options ie express-winston

**Comment by ZhiLinR** [comment]
me when wrong button

**Pull request merged**